### PR TITLE
created Dao for Company and Order

### PR DESCRIPTION
**Pull request by Dinesh05Patil**

### CODE REVIEW

**Review by Tejaprathap18 — APPROVED** [review]

File: src/main/java/com/ty/logestics/dao/OrderDao.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			return orderRepo.save(order);
+		} else {
+			return null;
+		}
+	}
+	
+	public Order deleteOrder(int id) {
+		if(orderRepo.findById(id).isPresent()) {
+			Order order=orderRepo.findById(id).get();
+			orderRepo.delete(order);
```
Comment (Narayana1308) [inline src/main/java/com/ty/logestics/dao/OrderDao.java:32]: use deleteById method

File: src/main/java/com/ty/logestics/dao/CompanyDao.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+			return companyRepo.save(company);
+		} else {
+			return null;
+		}
+	}
+
+	public Company deleteCompany(int id) {
+		if (companyRepo.findById(id).isPresent()) {
+			Company company = companyRepo.findById(id).get();
+			companyRepo.delete(company);
```
Comment (Narayana1308) [inline src/main/java/com/ty/logestics/dao/CompanyDao.java:33]: use deleteById method

**Review by Narayana1308 — CHANGES_REQUESTED** [review]
call the method deleteById instead of delete

File: src/main/java/com/ty/logestics/dao/CompanyDao.java (hunk 2 above)
Comment (Dinesh05Patil) [reply src/main/java/com/ty/logestics/dao/CompanyDao.java:33]: i am fetching company by id and deleting company object

**Review by Dinesh05Patil — COMMENTED** [review]
no changes required

**Review by Narayana1308 — APPROVED** [review]

**Review by PavaniBalabathuni — APPROVED** [review]

**Pull request merged**